src/components/Comments.jsx

```diff
@@ -12,8 +12,20 @@ function Comments({article_id}) {
     const [commentData, setCommentData] = useState({})
     const [commentPending, setCommentPending] = useState(false)
     const [refresh, setRefresh] = useState(0)
+    const [commentTooShort, setCommentTooShort] = useState(false)
+
+    useEffect(()=>{
```

Comment:
rather than setting this within a useEffect, it may be more pertinent to do this directly within the onChange where you set comment too short based on e.target.value so you don't have the delay that comes from the asynchronicity of setting new comment state then waiting to set the comment too short state.